modules/default/calendar/calendarfetcher.js

```diff
@@ -32,6 +32,7 @@ const CalendarFetcher = function (url, reloadInterval, excludedEvents, maximumEn
 
 	let reloadTimer = null;
 	let events = [];
+	let debug = false;
```

Comment:
Are those debug logs necessary for later? If yes, please use the logger js with the lowest level of logging